### ISSUE TITLE
Command argument after "-" is not visible in solarized dark theme

### ISSUE THREAD

**Issue by maidamai0** [JanDeDobbeleer/oh-my-posh2#229]
Given this command
```git
git commit -m "commit message" //became  git commit    "commit message"
git config --list // become git config
```
![image](https://user-images.githubusercontent.com/16829798/79414374-0533c400-7fdd-11ea-85bf-be5411ff2b32.png)

**Comment by maidamai0** [comment]
Works well on powershell, have this problem after I installed powershell 7.0 manuly.

**Comment by JanDeDobbeleer** [comment]
I'm running 7 as well but on MacOS, didn't see that yet. Will check again.

**Comment by myy1966** [comment]
When configuring PowerShell with the windows terminal, I encountered the same problem. Eventually, I found that the builtin solarized dark of the windows terminal is the same as the original color scheme released by Ethan Schoonover, and it has some bug for a long time. I replace the builtin solarized dark with a patched version that came from [https://github.com/mbadolato/iTerm2-Color-Schemes](https://github.com/mbadolato/iTerm2-Color-Schemes), and all is well.
